Automattic/wp-calypso
# Add thank you page to Jetpack Userless Checkout

#### Changes proposed in this Pull Request

* Add new thank-you screen exclusively for Jetpack ( "userless" ) Checkout purchases
* Modify composite checkout to use the jetpack site slug in all cases in the event of a Jetpack "userless" checkout
* Add a new shortcut `getProductName` selector.

![Untitled 12](https://user-images.githubusercontent.com/2810519/117895061-7abaa400-b272-11eb-8e43-358bad624373.png)
![Untitled 4](https://user-images.githubusercontent.com/2810519/117895072-7db59480-b272-11eb-8930-7032dd41c50c.png)


#### Testing instructions

1. Complete a test purchase with instructions for parent PR #51953
2. Verify that one purchase completion you are redirected to the thank-you page ( `/checkout/jetpack/thank-you` )
3. Verify the design matches the various examples above
